docs/src/pages/getting-started/faq/faq.md

```diff
@@ -23,23 +23,6 @@ There are many ways to support MUI:
   If you use MUI in a commercial project and would like to support its continued development by becoming a Sponsor, or in a side or hobby project and would like to become a Backer, you can do so through OpenCollective.
   All funds donated are managed transparently, and Sponsors receive recognition in the README and on the MUI home page.
 
-## Why aren't my components rendering correctly in production builds?
```

Comment:
I've moved the things related to v4 on the end and added the `[v4]` tag

Reply:
I think `[v4]` should be the prefix because it is easier to notice on global search.

Reply:
Fair enough, let me update